src/Agent/CHANGELOG.md

```diff
@@ -11,10 +11,14 @@ New Feature Description
 
 ### Fixes
 Fixes Issue [#224](https://github.com/newrelic/newrelic-dotnet-agent/issues/224): leading "SET" commands will be ignored when parsing compound SQL statements. ([#370](https://github.com/newrelic/newrelic-dotnet-agent/pull/370))
+
 Fixes Issue [#226](https://github.com/newrelic/newrelic-dotnet-agent/issues/226): the profiler ignores drive letter in `HOME_EXPANDED` when detecting running in Azure Web Apps. ([#373](https://github.com/newrelic/newrelic-dotnet-agent/pull/373))
 
+Fixes Issue [#93](https://github.com/newrelic/newrelic-dotnet-agent/issues/93): Update Async Timing Calculations ([#370](https://github.com/newrelic/newrelic-dotnet-agent/pull/374))
```

Comment:
Fix Issue [#93](https://github.com/newrelic/newrelic-dotnet-agent/issues/93): When the parent methods are blocked by their asynchronous child methods, the agent deducts the child methods' duration from parent methods' exclusive duration.([#374](https://github.com/newrelic/newrelic-dotnet-agent/pull/374))